src/Bundle/Resources/config/services.xml

```diff
@@ -43,9 +43,11 @@
                  class="Sylius\Bundle\ResourceBundle\Form\Extension\HttpFoundation\HttpFoundationRequestHandler"
                  decorates="form.type_extension.form.request_handler" decoration-priority="256" public="false" />
 
-        <service id="sylius.resource_registry" class="Sylius\Component\Resource\Metadata\Registry" public="false" />
+        <service id="sylius.resource_registry" class="Sylius\Resource\Metadata\Registry" public="false" />
         <service id="Sylius\Resource\Metadata\RegistryInterface" alias="sylius.resource_registry" public="false" />
-        <service id="Sylius\Component\Resource\Metadata\RegistryInterface" alias="sylius.resource_registry" public="false" />
+        <service id="Sylius\Component\Resource\Metadata\RegistryInterface" alias="sylius.resource_registry" public="false">
+            <deprecated package="sylius/resource-bundle" version="2.0">The "%alias_id%" service alias is deprecated. Use Sylius\Resource\Metadata\RegistryInterface instead.</deprecated>
```

Comment:
```suggestion
            <deprecated package="sylius/resource-bundle" version="1.11">The "%alias_id%" service alias is deprecated. Use Sylius\Resource\Metadata\RegistryInterface instead.</deprecated>
```
I think idea is to specify version since it was deprecated.

![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/34122b6b-4a50-4230-8295-1ecb3faf2239)

and if custom message is used, then how about like so,
```suggestion
            <deprecated package="sylius/resource-bundle" version="1.11">The "%alias_id%" service alias is deprecated since sylius/resource-bundle 1.11 and will be removed in sylius/resource-bundle 2.0. Use Sylius\Resource\Metadata\RegistryInterface instead.</deprecated>
```

Few examples from sylius/sylius

![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/20807a0e-9681-4f18-a2fe-9ae00311b25a)
![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/a941dbd1-1040-4d54-a310-3a4fc22bcceb)
![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/29141f93-f0ce-49e9-b728-aefc7474408e)